### PR TITLE
pqxtest: more docs

### PR DESCRIPTION
**Pull request by bmizerany**

### CODE REVIEW

File: pqxtest/pqxtest.go [hunk 1]
```diff
@@ -12,6 +13,119 @@
...
 //  	// NOTE: db will be closed automatically when t.Cleanup is called and the database will be dropped.
 //  }
+//
+// Developer Speed
+//
+// Pqxtest enables a developer to go from zero to well tested, production ready
+// code that interacts with a database in no time. This means removing much of
+// the toil assocaited with configuring and running a standalone postgres
+// instance, the can be brittle is left standing across test runs,
+// overenginnering with abudent interfaces for mocking.
```
Comment (bmizerany) [inline pqxtest/pqxtest.go]: Fix

File: pqxtest/pqxtest.go [hunk 2]
```diff
@@ -1,7 +1,8 @@
-// Package pqxtest provides functions for testing with a live, lightweight, standalone
+// Package pqxtest provides functions for testing with an "embeded" live, lightweight, standalone
```
Comment (pedro) [inline pqxtest/pqxtest.go]: typo, embedded. an hmm maybe no need for quotes?

File: pqxtest/pqxtest.go (hunk 1 above)
Comment (pedro) [inline pqxtest/pqxtest.go]: abundant 👌

File: pqxtest/pqxtest.go [hunk 3]
```diff
@@ -12,6 +13,119 @@
...
+// .github/workflows for an example.
+//
+// No Mocks
+//
+// Mocking database interactions is dangerous. It is easy to create a test that
+// gives false positives, or can cause develoers precious time hunting down a
+// bug that doesn't exist because they're getting false negatives from the mocks
+// (i.e. the code would work if running against an actual database).
+//
+// Writting mocks and mocking code is almost always avoiable when interacting
```
Comment (pedro) [inline pqxtest/pqxtest.go]: avoidable

File: pqxtest/pqxtest.go [hunk 4]
```diff
@@ -12,6 +13,119 @@
...
+// Some test libaries create a single transaction for a test and return a
+// wrapper-driver *sql.DB that runs all queries in a test in that trasaction.
+// This is less than ideal because, like mocks, this isn't how applications work
+// in production, so  it can lead to weird behaviors and false positives. It
+// also means you can't use those nifty driver specific functions you love.
+// Pqxtest returns a real *sql.DB.
+//
+// Speed
+//
+// Pqx is fast. It is designed to give you all the benfits of writting tests
```
Comment (pedro) [inline pqxtest/pqxtest.go]: benefits

File: pqxtest/pqxtest.go [hunk 5]
```diff
@@ -12,6 +13,119 @@
...
+// against a real database without slowing you down. That means no waiting for
+// setup and teardown after the first test run. The first "go test" takes only a
+// second or two to cache the standalone postgres binary and initialize the data
+// directory, but subsequent runs skip these steps, making them very fast!
```
Comment (pedro) [inline pqxtest/pqxtest.go:58]: truth! this is awesome

**Pull request merged**